textproc/heirloom-doctools/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        n-t-roff heirloom-doctools 160308
```

Comment:
The version should be set to something more meaningful. This looks like a date. So add a version in the ISO format YYYY-MM-DD. In this, I'm guessing it should be:

`version 2016-03-08
`